Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/TeamsController.cs

```diff
@@ -596,19 +584,9 @@ private async Task<List<ShiftsIntegResponse>> ProcessOpenShiftRequestApprovalAsy
                 {
                     this.telemetryClient.TrackTrace($"Shift mapping has failed for {finalOpenShiftRequest.Id}: " + ex.InnerException.ToString());
                 }
-                else
-                {
-                    this.telemetryClient.TrackTrace($"Shift mapping has resulted in some type of error with the following: {ex.StackTrace.ToString(CultureInfo.InvariantCulture)}");
-                }
 
-                // String should be coming from RESX file - Internal error occurred, saying that open shift request could be declined.
-                var error = new ResponseError()
-                {
-                    Code = Resource.InternalErrorCodeText,
-                    Message = Resource.InternalErrorCodeMessageText,
-                };
+                this.telemetryClient.TrackTrace($"Shift mapping has resulted in some type of error with the following: {ex.StackTrace.ToString(CultureInfo.InvariantCulture)}");
```

Comment:
You don't log the exception itself, or the message??

Reply:
Why do you log finalOpenShiftRequest.Id only when there's an inner exception?

Reply:
@aosolis - ack'd. Please see the screenshot below: 
![image](https://user-images.githubusercontent.com/5623947/76154092-4f935e00-608b-11ea-9ee7-ca487964a103.png)